lib/polist/struct.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Polist
+  module Struct
+    module ClassMethods
+      def struct(*attrs)
+        attr_accessor(*attrs)
+
+        define_method(:initialize) do |*args|
+          raise ArgumentError, "struct size differs" if args.length > attrs.length
```

Comment:
Error message does not match error condition

Reply:
That was taken straight from Ruby `Struct` class.

Reply:
Most inconsistent stdlib i ever met :(